### PR TITLE
item_count method added in DonorAdmin and DonationAdmin class in admi…

### LINKED ISSUE #154: Add item count on donations admin view indicating number of items
### Requirements
- Add item count on donations admin view indicating number of items

### PR DESCRIPTION
**Pull request by yuubd**
…n.py and refactored naming
closes #154

### CODE REVIEW

File: app/admin.py [hunk 1]
```diff
@@ -110,34 +110,41 @@ class DonorAdmin(admin.ModelAdmin):
...
     list_filter = ['want_receipt', 'province']
     search_fields = ['id', 'donor_name', 'telephone_number', 'mobile_number',
-                     'address_line', 'city', 'province', 'postal_code', 'customer_ref', 'email', ]
+                     'address_line', 'city', 'province', 'postal_code', 'customer_ref', 'email']
 
-    def get_donor(self, obj):
-        return obj.id
-    get_donor.short_description = 'Donor ID'
+    def item_count(self, obj):
+        count_of_donor = sum([donation.item_set.count() for donation in obj.donation_set.all()])
```
Comment (leesw98) [inline app/admin.py]: These aren't really `count_of_donor`. These are more like `count_per_donor`. Count of donor reminds of number of donors.

File: app/admin.py [hunk 2]
```diff
@@ -110,34 +110,41 @@ class DonorAdmin(admin.ModelAdmin):
...
-                    'verified')
-    readonly_fields = ('get_donation_donor_name',)
+                    'verified',
+                    'item_count')
+    readonly_fields = ('donor_name',)
     list_filter = ['pick_up', 'verified']
     search_fields = ['donor_id__donor_name', 'tax_receipt_no', 'donate_date', ]
 
-    def get_donation_donor_name(self, obj):
+    def donor_name(self, obj):
```
Comment (leesw98) [inline app/admin.py]: Clean 👍

**Comment by leesw98** [comment]
I approved CC issues. It seemed trivial enough.

**Comment by yuubd** [comment]
Great, I will close the corresponding issue when it is merged.

**Pull request merged**